packages/@sanity/form-builder/src/inputs/File/FileInput.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable complexity */
```

Comment:
Why is this needed?

Reply:
linter underlines everything and makes it hard to read (complexity warning). We should consider change the eslint settings.

Reply:
Yeah, I noticed afterward that warning has started to appear in deeply nested jsx now. Looks like a bug in eslint, actually. Just leave it for now :)